zebra/zebra_vty.c

```diff
@@ -2566,39 +2563,64 @@ DEFUN (default_vrf_vni_mapping,
 	if (argc == 3)
 		filter = 1;
 
-	ret = zebra_vxlan_process_vrf_vni_cmd(zvrf, vni, err, ERR_STR_SZ,
-					      filter, 1);
-	if (ret != 0) {
-		vty_out(vty, "%s\n", err);
-		return CMD_WARNING;
-	}
+	nb_cli_enqueue_change(
+		vty, "/frr-vrf:lib/vrf[name='default']/frr-zebra:zebra",
+		NB_OP_CREATE, NULL);
```

Comment:
if zebra is called with -o option, then that means the default vrf name may be somethng else than default.
without entering in the specificities of -o option ( and findout more bugs), can you make generic the string 'default' by forging the string before calling nb_cli_enqueue_change , please ?

Reply:
Fixed.